src/components/molecules/Common/AssetModal/AssetContainer/index.tsx

```diff
@@ -97,7 +98,10 @@ const AssetContainer: React.FC<Props> = ({
     onRemove,
     onSearch,
   });
-
+  const wrapperRef = useRef<HTMLDivElement>(null);
+  useEffect(() => {
+    if (wrapperRef.current && !isLoading && hasMoreAssets) AutoFetchMore(wrapperRef, onGetMore);
+  }, [hasMoreAssets, isLoading, onGetMore]);
```

Comment:
Please think about how things are spaced out. We don't have a definitive rule because the types of files/components are various, but trying to space things well is important for readability. In this case, useT, useHooks and useRef are all declaring variables being used elsewhere, so is typically fine side by side, but useEffects are unique and I think need to stand on their own so it is easier to spot and acknowledge.
Hope that makes sense. Please ask me on slack if you want more of an explanation or you want to debate it!
```suggestion
  const wrapperRef = useRef<HTMLDivElement>(null);
  
  useEffect(() => {
    if (wrapperRef.current && !isLoading && hasMoreAssets) AutoFetchMore(wrapperRef, onGetMore);
  }, [hasMoreAssets, isLoading, onGetMore]);
  
```